api/src/pcapi/core/finance/models.py

```diff
@@ -1055,6 +1055,30 @@ def isClosed(cls) -> Exists:  # pylint: disable=no-self-argument
             .where(Cashflow.status == CashflowStatus.ACCEPTED)
         )
 
+    @property
+    def cashflow_batch_label(self) -> str | None:
+        for booking_finance_incident in self.booking_finance_incidents:
+            for finance_event in booking_finance_incident.finance_events:
+                for pricing in finance_event.pricings:
+                    if pricing.status in (PricingStatus.PROCESSED, PricingStatus.INVOICED):
+                        for cashflow in pricing.cashflows:
+                            return cashflow.batch.label
```

Comment:
on pourrait pas remplacer ce for par ça ?
```suggestion
                        if pricing.cashflows:
                            return pricing.cashflows[0].batch.label
```
ça rendrait la pile de for un peu moins effreyante